package.json

```diff
@@ -12,14 +12,19 @@
     "@discordjs/builders": "^0.11.0",
     "@discordjs/rest": "^0.2.0-canary.0",
     "@types/js-yaml": "^4.0.5",
+    "@types/katex": "^0.14.0",
     "@types/node": "^16.11.12",
     "axios": "^0.27.2",
     "bufferutil": "^4.0.5",
     "discord-api-types": "^0.26.0",
     "discord.js": "^13.5.0",
     "fuse.js": "^6.5.3",
+    "html-to-image": "^1.9.0",
+    "html2canvas": "^1.4.1",
```

Comment:
Don't mind about these two, I will remove them if this feature works.

Reply:
See comment below about leaving things for "if it works".